vue/components/ui/molecules/filter/filter.vue

```diff
@@ -334,6 +363,30 @@ export const Filter = {
                 return false;
             }
 
+            if (auto) {
+                const oldItems = this.items.map(item => item.id);
+                const newItems = items.map(item => item.id);
+                const freshItems = newItems.reduce((a = 0, b) => a + !oldItems.includes(b), 0);

```

Comment:
This is confusing, I would add a comment explaining what happens,

Reply:
I think we just use inline comments, no need for `@param`, just add a small text explaining whats going on